crates/subspace-runtime-primitives/src/lib.rs

```diff
@@ -78,6 +78,9 @@ pub type Hash = sp_core::H256;
 /// Type used for expressing timestamp.
 pub type Moment = u64;
 
+/// Type used to represent a FeedId or ChainId
+pub type FeedId = u64;
```

Comment:
This crate is for data structures shared between both of the runtimes we have. `FeedId` is only used in one of them and thus should probably be added to the runtime where it is used instead.

Reply:
That would lead to double defining these types since we would also need to use them test runtimes as well.
We can add this to either `subspace-core-primitives` or `subspace-runtime-primitives`. What is the concern here ?

Reply:
The concern is only conceptual, this crate was created for specific purpose of sharing data structures between primary and secondary runtime, not to share data structures between production and test runtimes and primarily for data structures or constant inherent to the consensus or blockchain as such. This is just a local data structure for one pallet that doesn't even have to be the same in test runtime.

Reply:
Sure but this will only lead to duplicate types for the runtimes. hence, I was suggesting we have a primitives crate common for both